lib/Service/MailTransmission.php

```diff
@@ -142,8 +149,12 @@ public function sendMessage(Account $account, LocalMessage $localMessage): void
 			$mail->send($transport, false, false);
 			$localMessage->setRaw($mail->getRaw(false));
 		} catch (Horde_Mime_Exception $e) {
-			$localMessage->setStatus(LocalMessage::STATUS_SMPT_SEND_FAIL);
 			$this->logger->error($e->getMessage(), ['exception' => $e]);
+			if (in_array($e->getCode(), $this->retriableCodes)) {
```

Comment:
```suggestion
			if (in_array($e->getCode(), $this->retriableCodes, true)) {
```

Install https://plugins.jetbrains.com/plugin/7622-php-inspections-ea-extended-